### PR TITLE
[JENKINS-37324] Add an ArgumentsColumn to the FlowGraphTable display

### PR DESCRIPTION
**Pull request by jglick**
screenshot

See comment in [JENKINS-37324](https://issues.jenkins-ci.org/browse/JENKINS-37324). Makes sense to use the new system in the rudimentary built-in UI.

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/ArgumentsColumn.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import hudson.Extension;
+import org.jenkinsci.plugins.workflow.actions.ArgumentsAction;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+import org.jenkinsci.plugins.workflow.visualization.table.FlowNodeViewColumn;
+import org.jenkinsci.plugins.workflow.visualization.table.FlowNodeViewColumnDescriptor;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+@Restricted(NoExternalUse.class)
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/ArgumentsColumn.java]: I am not sure if it makes the descriptor restricted. Likely no

Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/ArgumentsColumn.java]: If not, I guess that would be considered a bug in the access modifier restrictions library.

**Comment by jglick** [comment]
@reviewbybees done

**Comment by svanoort** [comment]
Love it!

**Pull request merged**